spec/realworld/parallel_spec.rb

```diff
@@ -6,7 +6,7 @@
       source "https://rubygems.org"
       gem 'activesupport', '~> 3.2.13'
       gem 'faker', '~> 1.1.2'
-      gem 'i18n', '~> 0.6.0' # Because 1.7+ requires Ruby 1.9.3+
+      gem 'i18n', '~> 0.7.0' # Because 1.7+ requires Ruby 1.9.3+
```

Comment:
If you have to change the tests for your refactor to pass the tests, I think that means the refactor doesn't work. :) Please make sure these tests pass without changing them!